src/Language/PureScript/Publish.hs

```diff
@@ -230,7 +230,7 @@ checkLicense pkgMeta =
 -- Check if a string is a valid SPDX license expression.
 --
 isValidSPDX :: String -> Bool
-isValidSPDX = (== 1) . length . SPDX.parseExpression
+isValidSPDX = isJust . SPDX.mkLicenseId
```

Comment:
As I understand it, this will attempt to parse a license ID rather than a license expression, which means that e.g. "GPL-2.0+", meaning GPL 2.0 or any later version, would not be accepted. From looking at the docs, I think we need to use the `Parsec` instance for `License`, and then check that a) the parse succeeds and b) that the result is not `NONE`.

Reply:
I see, thanks for checking. How about now?

Reply:
Looks good now, thanks!